From 880c6d51195f68e63686a1bf814d19d4ad112686 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Petr=20P=C3=ADsa=C5=99?= <ppisar@redhat.com>
Date: Wed, 7 Mar 2018 14:45:29 +0100
Subject: [PATCH 1/3] Fix compiler warnings about unused variables and implicit
 declarations
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

This patch fixes these warnings:

src/xmlparser.c:178:4: warning: implicit declaration of function ‘LOG’
src/xmlparser.c:298:7: warning: unused variable ‘this_mac_p’
src/xmlparser.c:296:8: warning: unused variable ‘mac_string_p’
src/xmlparser.c:296:20: warning: unused variable ‘this_mac’

Signed-off-by: Petr Písař <ppisar@redhat.com>
---
 src/log.h       | 1 +
 src/xmlparser.c | 3 ---
 2 files changed, 1 insertion(+), 3 deletions(-)

diff --git a/src/log.h b/src/log.h
index 0add969..ee3460b 100644
--- a/src/log.h
+++ b/src/log.h
@@ -24,5 +24,6 @@
 
 
 FILE *log_file;
+void LOG(const char *fmt, ...);
 
 #endif
diff --git a/src/xmlparser.c b/src/xmlparser.c
index 2a8f0b4..3a5052e 100644
--- a/src/xmlparser.c
+++ b/src/xmlparser.c
@@ -293,9 +293,6 @@ u08 extractMacAddress(char *mac_string, apr_hash_t **table, apr_pool_t *pool)
 u08 extractRuleMac(xmlNode *xml_rule, struct rule *this_rule)
 {
 	char *mac_string = NULL;
-	char *mac_string_p = NULL;
-	struct macaddress this_mac;
-	u08 *this_mac_p = &(this_mac.cell[0]);
 
 	mac_string = (char *)xmlGetProp(xml_rule, (xmlChar *)"mac");
 	if(mac_string != NULL)
-- 
2.13.6

